main.cpp

```diff
@@ -566,6 +567,7 @@ int main(int argc, char *argv[])
 	cbs.add("--cpp", [&args](CLIParser &) { args.cpp = true; });
 	cbs.add("--cpp-interface-name", [&args](CLIParser &parser) { args.cpp_interface_name = parser.next_string(); });
 	cbs.add("--metal", [&args](CLIParser &) { args.metal = true; });
+	cbs.add("--msl-no-pack-ubos", [&args](CLIParser &) { args.msl_pack_ubos = false; });
```

Comment:
...so should we change "metal" to "msl" instead?

So...

	./test_shaders.py --msl shaders-msl

instead of

	./test_shaders.py --metal shaders-msl

and then

	./test_shaders.py  --msl shaders-msl --msl-no-pack-ubos